portfolio/src/main/webapp/script.js

```diff
@@ -15,6 +15,7 @@
 /**
  * Adds a random greeting to the page.
```

Comment:
let's update this comment and delete the commented out addRandomGreeting() method